### PR TITLE
refactor: i18n translations (part-1)

### PR DESCRIPTION
**Pull request by dstala**
## Change Summary
refactor: i18n translations for project menu

## Change type
- [x] refactor: (refactoring production code, eg. renaming a variable)

## Test/ Verification
Locally verified

## Additional information / screenshots (optional)
associated master spreadsheet (for GUI v2)
https://docs.google.com/spreadsheets/d/1hvyBRzDOcEVzfUq6Ngbl70UCnIoccR5nCo2XznO_qKk/edit#gid=2076107172

### CODE REVIEW

**Comment by dstala** [comment]
@wingkwong please refer to this sheet
https://docs.google.com/spreadsheets/d/1hvyBRzDOcEVzfUq6Ngbl70UCnIoccR5nCo2XznO_qKk/edit#gid=2076107172

newly introduced items are placed at end of file.

File: packages/nc-gui-v2/lang/zh_CN.json [hunk 1]
```diff
@@ -238,7 +240,12 @@
     "childTable": "子表",
     "childColumn": "子列",
     "onUpdate": "更新",
-    "onDelete": "删除"
+    "onDelete": "删除",
+    "account": "帐户",
+    "language": "语",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_CN.json:245]: 语言

File: packages/nc-gui-v2/lang/zh_CN.json [hunk 2]
```diff
@@ -238,7 +240,12 @@
...
+    "primaryColor": "原色",
+    "accentColor": "口音颜色",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_CN.json:247]: 强调色

File: packages/nc-gui-v2/lang/zh_CN.json [hunk 3]
```diff
@@ -238,7 +240,12 @@
     "childTable": "子表",
     "childColumn": "子列",
     "onUpdate": "更新",
-    "onDelete": "删除"
+    "onDelete": "删除",
+    "account": "帐户",
+    "language": "语",
+    "primaryColor": "原色",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_CN.json:246]: 主色调

File: packages/nc-gui-v2/lang/zh_HK.json [hunk 4]
```diff
@@ -238,7 +240,12 @@
     "childTable": "兒童表",
     "childColumn": "兒童專欄",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_HK.json:241]: 子列

File: packages/nc-gui-v2/lang/zh_HK.json [hunk 5]
```diff
@@ -238,7 +240,12 @@
     "childTable": "兒童表",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_HK.json:240]: 子表

File: packages/nc-gui-v2/lang/zh_HK.json [hunk 6]
```diff
@@ -238,7 +240,12 @@
...
     "childColumn": "兒童專欄",
     "onUpdate": "更新",
-    "onDelete": "在刪除"
+    "onDelete": "在刪除",
+    "account": "帳戶",
+    "language": "語",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_HK.json:245]: 語言

File: packages/nc-gui-v2/lang/zh_HK.json [hunk 7]
```diff
@@ -238,7 +240,12 @@
...
+    "primaryColor": "原色",
+    "accentColor": "口音顏色",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_HK.json:247]: 強調色

File: packages/nc-gui-v2/lang/zh_HK.json [hunk 8]
```diff
@@ -238,7 +240,12 @@
     "childTable": "兒童表",
     "childColumn": "兒童專欄",
     "onUpdate": "更新",
-    "onDelete": "在刪除"
+    "onDelete": "在刪除",
+    "account": "帳戶",
+    "language": "語",
+    "primaryColor": "原色",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_HK.json:246]: 主色調

File: packages/nc-gui-v2/lang/zh_TW.json [hunk 9]
```diff
@@ -238,7 +240,12 @@
     "childTable": "子表格",
     "childColumn": "子欄",
     "onUpdate": "更新",
-    "onDelete": "在刪除"
+    "onDelete": "在刪除",
+    "account": "帳戶",
+    "language": "語",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_TW.json:245]: 語言

File: packages/nc-gui-v2/lang/zh_TW.json [hunk 10]
```diff
@@ -238,7 +240,12 @@
...
+    "primaryColor": "原色",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_TW.json:246]: 主色調

File: packages/nc-gui-v2/lang/zh_TW.json [hunk 11]
```diff
@@ -238,7 +240,12 @@
...
+    "accentColor": "口音顏色",
```
Comment (wingkwong) [inline packages/nc-gui-v2/lang/zh_TW.json:247]: 強調色

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Pull request merged**